docs/contributing_dev.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Contributing to DEV
```

Comment:
As long as we're creating a whole new permalink we might reference later, I wonder if we'd want to name it `contributing_forem` and `Contributing to Forem`.

This will become more natural soon and we don't necessarily need to rush _all our copy_, but as long as we're creating new URLs, maybe adopt that?

Reply:
Done!